SupportClient/SupportClient/Views/Cells/FeedbackCollectionViewCell.swift

```diff
@@ -0,0 +1,103 @@
+//
+//  FeedbackCollectionViewCell.swift
+//  SupportClient
+//
+//  Created by Yana Sang on 5/8/20.
+//  Copyright © 2020 Cornell Appdev. All rights reserved.
+//
+
+import UIKit
+
+enum FeedbackSection {
+    case customerService
+    case bugsAndRequests
+}
+
+class FeedbackCollectionViewCell: UICollectionViewCell {
+
+    static let reuseIdentifier = "feedbackCollectionViewCell"
+
+    private var feedbackSection: FeedbackSection = .customerService
+    private let emptyStateView = NoResultsEmptyStateView()
+    private var items: [Feedback] = []
+    private let tableView = UITableView()
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+
+        setupViews()
+        setupConstraints()
+    }
+
+    func setupViews() {
+        tableView.delegate = self
+        tableView.dataSource = self
+        tableView.tableFooterView = UIView()
+        tableView.register(BugsRequestsTableViewCell.self, forCellReuseIdentifier: BugsRequestsTableViewCell.reuseIdentifier)
+        tableView.register(CustomerServiceTableViewCell.self, forCellReuseIdentifier: CustomerServiceTableViewCell.reuseIdentifier)
+        tableView.translatesAutoresizingMaskIntoConstraints = false
+        contentView.addSubview(tableView)
+
+        emptyStateView.isHidden = true
+        emptyStateView.translatesAutoresizingMaskIntoConstraints = false
+        contentView.addSubview(emptyStateView)
+    }
+
+    func setupConstraints() {
+        NSLayoutConstraint.activate([
+            tableView.leadingAnchor.constraint(equalTo: contentView.leadingAnchor),
+            tableView.topAnchor.constraint(equalTo: contentView.topAnchor),
+            tableView.trailingAnchor.constraint(equalTo: contentView.trailingAnchor),
+            tableView.bottomAnchor.constraint(equalTo: contentView.bottomAnchor)
+        ])
+
+        NSLayoutConstraint.activate([
+            emptyStateView.centerXAnchor.constraint(equalTo: contentView.centerXAnchor),
+            emptyStateView.topAnchor.constraint(equalTo: contentView.topAnchor, constant: 118)
+        ])
+    }
+
+    func configure(section: FeedbackSection, items: [Feedback]) {
+        self.feedbackSection = section
+        self.items = items
+        self.emptyStateView.isHidden = !items.isEmpty
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+}
+
+extension FeedbackCollectionViewCell: UITableViewDelegate {
+
+    func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
+        return 103.6
```

Comment:
sry v small nit n this is prob more for ravina to keep in mind but can we round this u p to 104. And whichever constraint that is causing the .6 we can round up to 1. Not a huge deal tho if u dont wanna fix rn